x/btcstaking/keeper/power_dist_change.go

```diff
@@ -150,15 +150,14 @@ func (k Keeper) ProcessAllPowerDistUpdateEvents(
 	ctx context.Context,
 	dc *types.VotingPowerDistCache,
 	events []*types.EventPowerDistUpdate,
-	maxActiveFps uint32,
```

Comment:
Is this max being enforced somewhere else? Just curious.

Reply:
Yep, here https://github.com/babylonlabs-io/babylon/blob/0a90a5e404fecdd0650bc0042ffff6735e6f515b/x/btcstaking/keeper/power_dist_change.go#L75.